packages/components/src/GridLayout/Dashboard.stories.js

```diff
@@ -133,49 +134,49 @@ function TileWithAction({ tile }) {
 	);
 }
 
-function GridContainer({ isLoading = false, skeletonConfiguration, isResizable = true, ...rest }) {
-	const [tiles, setTiles] = useState([
```

Comment:
This was added as a state but it's a constant, never changed. I moved it outside of the component